es-space-mgmt/main/default/aura/spaceDesignerAura/spaceDesignerAuraController.js

```diff
@@ -3,24 +3,29 @@
         var recId = event.getParam('reservationId');
         var market = event.getParam('marketId');
         var customer = event.getParam('customerName');
+
         component.set('v.reservation', recId);
         component.set('v.customer', customer);
         helper.startFlow(component, recId, market);
     },
 
-    handleReset: function (component, event, helper) {
+    handleReset: function (component, event) {
         if (event.getParam('status') === 'FINISHED') {
             var spaceDesignerLWC = component.find('spaceDesignerLWC');
+
             spaceDesignerLWC.handleFlowExit();
             var navigationItemAPI = component.find('navigationItemAPI');
+
             navigationItemAPI
                 .refreshNavigationItem()
                 .then(function (response) {
```

Comment:
Same comment as for `reservationHelperAuraHelper.js`: drop the logging block or show a toast?